monkey/infection_monkey/pe/actions/tools.py

```diff
@@ -0,0 +1,68 @@
+'''
+Contains functions helpful for pe module
```

Comment:
I feel like there is a better way to achieve shared code besides having a `tools.py` files in all of our folders. @itaymmguardicore @ShayNehmad any ideas?